playbooks/templates/.env.j2

```diff
@@ -268,6 +268,17 @@ ABUSE_LOG_LEVEL={{ webportal_common_config.abuse_log_level }}
 # REQUIRED: Hardcoded, not configurable
 ABUSE_SPONSOR="Abuse Scanner"
 
+# Reporting related environment variables
+#
+# OPTIONAL: manually defined by user, defaults to false
+ABUSE_NCMEC_REPORTING_ENABLED={{
+webportal_common_config.abuse_ncmec_reporting_enabled | default('false') }}
+#
+# REQUIRED: manually defined by user, only if reporting is enabled
```

Comment:
REQ: A user could have the abuse scanner enabled but not the NCMEC reporting correct?

If so then these should be in an if condition based on `webportal_common_config.abuse_ncmec_reporting_enabled` otherwise ansible is going to complain and error when these aren't defined. 
```
{% if webportal_common_config.abuse_ncmec_reporting_enabled %}
...
{% endif %}

Reply:
right, thanks!